docs/general/ddl/replication.rst

```diff
@@ -1,73 +1,100 @@
-.. _replication:
+.. _ddl-replication:
 
 ===========
 Replication
 ===========
 
-Replication of a table in CrateDB means that each primary shard of a table is
-stored additionally on so called secondary shards. This can be useful for
-better read performance and high availability.
+You can configure CrateDB to *replicate* tables. When you configure
+replication, CrateDB will try to ensure that every table :ref:`shard
+<ddl-sharding>` has one or more copies available at all times.
 
-If not specified, CrateDB creates zero to one replica depending on the number
-of available nodes at the cluster. At a single-node cluster, replicas are set
-to zero to allow fast write operations with the default setting of
-:ref:`sql-create-table-write-wait-for-active-shards`.
+When there are multiple copies of the same shard, CrateDB will mark one as the
+*primary shard* and treat the rest as *replica shards*. Write operations
+always go to the primary shard, whereas read operations can go to any
+shard. CrateDB continually synchronizes data from the primary shard to all
+replica shards (through a process known as :ref:`shard recovery
+<gloss-shard-recovery>`).
 
-.. rubric:: Table of contents
+When a primary shard is lost (e.g., due to node failure), CrateDB will promote
+a replica shard to primary. Hence, more table replicas mean a smaller chance of
+permanent data loss (through increased `data redundancy`) in exchange for more
+disk space utilization and intra-cluster network traffic.
 
-.. contents::
-    :local:
+CrateDB :ref:`allocates <gloss-shard-allocation>` every shard to a specific
+node. CrateDB will try to allocate the primary shard and replica shards to
+different nodes to maximize :ref:`durability <concept-durability>` and
+:ref:`resiliency <concept-resiliency>`.
```

Comment:
```suggestion
node. CrateDB will enforce the allocation of the primary shard and replica 
shards on different nodes to maximize :ref:`durability <concept-durability>`
and :ref:`resiliency <concept-resiliency>`.
If not enough nodes are available for each replica shard to be allocated,
the table will go into a `YELLOW` health state, see :ref:`sys.health <sys-health>`.
```

Reply:
ty!!

I reworded slightly::

    CrateDB :ref:`allocates <gloss-shard-allocation>` every shard to a specific
    node. Normally, CrateDB will enforce the requirement that the primary shard and
    replica shards are each allocated to a different node. If that requirement
    cannot be satisfied, the cluster will be given a *yellow* :ref:`health status
    <sys-health>`.

Reply:
gotcha. thanks!